ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/CompletionStreamElement.scala

```diff
@@ -12,6 +12,7 @@ object CompletionStreamElement {
 
   final case class CheckpointElement(checkpoint: Checkpoint) extends CompletionStreamElement
 
-  final case class CompletionElement(completion: Completion) extends CompletionStreamElement
+  final case class CompletionElement(checkpoint: Option[Checkpoint], completion: Completion)
```

Comment:
You are referencing these two items in the `completion` -> `checkpoint` order everywhere except here, hence, for consistency it makes sense to make this case class follow the previous order as well.

Reply:
Done